README.md

```diff
@@ -233,7 +238,7 @@ For example, in VS Code, click `Debug` > `Add Configuration...` > `Java: Attach`
 
 6. When you are done, press Ctrl-C in the console to terminate dev mode and stop your server.
 
-## VS Code extension
+## VS Code extension (Currently maven only)
```

Comment:
Remove ` (Currently maven only)` from the title and add another section after the first paragraph instead, e.g. `At the moment, this extension is for Maven projects only.`